data/map planets.txt

```diff
@@ -3061,6 +3080,12 @@ planet "New Kansas"
 		threshold 2500
 		fleet "Small Militia" 22
 
+planet "New Kola"
+	attributes uninhabited
+	landscape land/badlands7
+	description `Although from outside New Kola appears to be a gas giant, its thick, dense atmosphere obscures the rocky surface of a huge super-earth. Even if it were made of breathable gases, the pressure at the surface is beyond what humans can tolerate - only at the peaks of the highest mountains can you safely step outside wearing only oxygen gear, and even then the crushing gravity would cause serious problems if you stayed for long.`
```

Comment:
If the air pressure is broadly tolerable, so is the gravity.

Reply:
I'll admit I'm not particularly well-versed in planetary geography, but what I'm trying to say here is that the air pressure is only tolerable on high mountain peaks, not at ground level.
To the best of my knowledge though, there's quite a large degree of variability in atmospheric pressure relative to gravity - see Earth vs Venus, or Titan vs Ganymede.

Reply:
Air pressure is proportional to gravity. If there is a place where you can land and tolerate the air pressure pretty well (without also needing protective gear because it's corrosive or whatnot), then _at that place_ you can also tolerate the gravity pretty well.

Titan has similar pressure to earth because it's atmosphere is made up of much heavier stuff, and is super cold.
Venus atmosphere has much greater pressure than earth because it's made of of much heavier stuff, and is super corrosive.

If you can safely step outside with _just_ oxygen gear at a particular level in the atmosphere, you can also tolerate the gravity at the same level as well.

Reply:
> Air pressure is proportional to gravity. If there is a place where you can land and tolerate the air pressure pretty well (without also needing protective gear because it's corrosive or whatnot), then at that place you can also tolerate the gravity pretty well.

I don't think that's right. At the top of Mount Everest, the atmospheric pressure is about one third of that at sea level, and the gravity is almost exactly the same as at sea level. To take a more extreme example, the gravitational pull experienced at the altitude of low earth orbit is less than 10% less than at sea level, but the atmospheric pressure is zero.
As far as I can tell, atmospheric pressure is the result of the weight (i.e. mass times gravity), of all the atmosphere above a particular point, so it would be increased by increasing the mass of the atmosphere or by increasing the strength of gravity. But as you move up, an increasing portion of the atmosphere is now below you rather than above, and hence isn't weighing down on the atmosphere where you are.

Reply:
And at the top of Everest you need more than just oxygen gear because it's cold enough to kill you, and quite quickly.

> If you can safely step outside with **_JUST_** oxygen gear at a particular level in the atmosphere, you can also tolerate the gravity at the same level as well.

Reply:
Sure, but a thicker atmosphere would capture more heat. I imagine all other things being equal, you'd expect that temperatures at the altitude where the atmospheric pressure is about 1atm would be about the same whether that's ground level or it's the top of a mountain and there's ten kilometers more atmosphere underneath you. Also, this atmosphere has a different composition to Earth's, which would also have effects on the surface temperature.

Reply:
> I imagine all other things being equal, you'd expect that temperatures at the altitude where the atmospheric pressure is about 1atm would be about the same whether that's ground level or it's the top of a mountain and there's ten kilometers more atmosphere underneath you.
You would imagine wrongly. For example, Jupiter's atmosphere is -108C (-163F) where the pressure is 1atm. Saturn's is -138C. Uranus -197C.
Even Venus, which has a cloud layer at approximately 1atm that would be tolerable to humans is 30C compared to Earth's average 15C. And that particular layer is relatively nice gasses like nitrogen and carbon dioxide.

Temperature, composition, pressure, and atmosphere volume are all inputs into a multivariable function where we survive in a very narrow window.

If you can step outside in your everyday cotton flight suit or whatever, and JUST use oxygen gear, you can tolerate the gravity.

Reply:
> You would imagine wrongly. For example, Jupiter's atmosphere is -108C (-163F) where the pressure is 1atm. Saturn's is -138C. Uranus -197C.

I did say "All other things being equal". Jupiter, Saturn and Uranus orbit far outside the habitable zone, and recieve far less sunlight. That's not "all other things being equal", because solar insolation is an "other thing" and is not equal.

Conversely, Venus is closer to the sun and recieves more sunlight, though I'm a little surprised by how little difference that makes at the 1atm layer.

> Temperature, composition, pressure, and atmosphere volume are all inputs into a multivariable function where we survive in a very narrow window.

You're correct that these are all inputs into a function, and that we can only survive in a narrow window, but that doesn't mean only a narrow range of combinations of conditions is possible. Gravity is determined by planetary mass and density (or radius and density, or mass and radius - any two determine the third), largely independent of the atmosphere. Atmospheric pressure is then determined based on that and on the total amount of atmosphere present, which, as I pointed out, can vary massively for a given planetary mass. (plot Titan, Ganymede, Earth, Venus, and Mars on a graph, and what you certainly don't get is a nice line). Temperature then can be varied with no effect on gravity and little effect on total atmosphere (as long as gases don't start freezing out) based on the amount of light recieved from the star. 
With all that in mind, I don't think the combination of characteristics here is impossible - It's just difficult to produce a concrete proof because our solar system lacks any terrestrial planets larger than earth. I think it's pretty plausible that a super-earth type terrestrial planet, with an atmosphere somewhere between Earth's and Venus', would be pretty much exactly as I describe New Kola.

Reply:
For the love of god I don't have the time or the crayons for this, but at the _very least_ you need some protection from wind chill to otherwise be there with only breathing gear.

Reply:
Insulting people's intelligence isn't necessary, regardless of how much in the right you feel you are.

Reply:
Gravity and wind chill are two separate things though. If you want to argue that it would be *too cold* to step outside with only oxygen gear, that's one thing, but that's different from your previous argument that the air pressure would necessarily be intolerable.

Reply:
It's a combination of things. With the given combination as it stands, either the temp is cold and wind chill will freeze you, or the temp is warm and the wind will suck the moisture from you and you'll die of dehydration quicker than the gravity will be an issue.
And even that possibility is falling into an extremely narrow range of only hypothetically plausible possibilities.

> air pressure would necessarily be intolerable.

And I didn't say it would *necessarily* be intolerable, I said that *either* the pressure would be intolerable, or the atmosphere would kill you other ways - which remains consistent with my point.